package.json

```diff
@@ -2,29 +2,17 @@
   "name": "courseroad",
   "version": "1.0.0",
   "description": "Course planning for MIT undergrads",
-  "main": "index.js",
+  "author": "npfoss, cvorbach, ksmori",
   "scripts": {
-    "lint": "node_modules/.bin/eslint --ext .js,.vue src/",
-    "lint-fix": "node_modules/.bin/eslint --ext .js,.vue --fix src/",
     "test": "test",
+    "lint": "node_modules/.bin/eslint --ext .js,.vue src/",
     "build-dev": "webpack --env.APP_URL=https://courseroad.mit.edu/dev --env.FIREROAD_URL=https://fireroad.mit.edu --mode=production --config build/webpack.config.dev.js",
     "build-prod": "webpack --env.APP_URL=https://courseroad.mit.edu --env.FIREROAD_URL=https://fireroad.mit.edu --mode=production --config build/webpack.config.dev.js",
-    "dev": "webpack-dev-server --env.APP_URL=http://localhost:8080 --env.FIREROAD_URL=https://fireroad-dev.mit.edu --mode=development --config build/webpack.config.dev.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/npfoss/courseroad2.git"
-  },
-  "keywords": [
-    "mit",
-    "courseroad"
-  ],
-  "author": "npfoss, cvorbach, ksmori",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/npfoss/courseroad2/issues"
+    "dev": "webpack-dev-server --env.APP_URL=http://localhost:8080 --env.FIREROAD_URL=https://fireroad-dev.mit.edu --mode=development --config build/webpack.config.dev.js",
+    "lint-fix": "node_modules/.bin/eslint --ext .js,.vue --fix src/",
```

Comment:
why did you move these around? I feel like the two lint commands should be together